packages/@aws-cdk/aws-route53/test/vpc-endpoint-service-domain-name.test.ts

```diff
@@ -33,7 +34,7 @@ beforeEach(() => {
   });
 });
 
-test('create domain name resource', () => {
+test.skip('create domain name resource', () => {
```

Comment:
`!` This just means the test is effectively dead. It's an unfortunate reality that every module where we've upgraded to jest we've found at least one test that is broken. Did you try to troubleshoot it? What's the error? An alternative would be to postpone the move to jest by pulling this out into its own PR, but as-is, I'd much rather we fix this test than skip it.

Reply:
Unfortunately this test contains a lots of assertions for resources which I am not sure are related or not. I will try to fix this if possible.

Reply:
Fixed the tests.